src/components/CandleCharts.js

```diff
@@ -0,0 +1,83 @@
+import { VictoryChart,VictoryCandlestick,VictoryAxis,VictoryTheme,VictoryZoomContainer,VictoryScatter,VictoryCursorContainer } from "victory";
+import { useEffect,useState } from "react";
+const CandleCharts=()=>{
+
+
+  const axios = require('axios')
+  const baseUrl = "https://api-pub.bitfinex.com/v2/";
+  
+  const queryParams = "limit=150"    
+
+    const [value,setValue]=useState([])
+    const [series,setSeries]=useState([]);
+    const [duration,setDuration]=useState('12h')
+    const [zoom,setZoom]=useState()
+    const [cursor,setCursor]=useState()
+
+    const pathParams = `candles/trade:${duration}:tBTCUSD/hist`
+
+    useEffect(()=>{
+        axios.get(`${baseUrl}/${pathParams}?${queryParams}`)
```

Comment:
create separate file for API call

Reply:
here i am making a call with dynamic parameters.Should i still do that?